src/common/util.cc

```diff
@@ -274,7 +274,7 @@ string cleanbin(bufferlist &bl, bool &base64)
 {
   bufferlist::iterator it;
   for (it = bl.begin(); it != bl.end(); ++it) {
-    if (iscntrl(*it))
+    if (iscntrl(*it) && !isspace(*it))
```

Comment:
@dzafman i see. and i am re-reading how the `cleanbin()` is being used in cot. seems we are not using it to print out the json/xml elements. so i think it should be fine. =)